blog/2023-05-25-tigris-standalone-search/index.mdx

```diff
@@ -0,0 +1,528 @@
+---
+slug: tigris-standalone-search
+title: Enabling awesome search experiences with Tigris Standalone Search
+description: >
+  Enabling awesome search experiences with Tigris Standalone Search
+keywords: [nosql search, standalone search, search engine]
+authors: [leggetter]
+tags: [nosql search, standalone search]
+image: "./tigris-standalone-search.png"
+---
+
+<head>
+  <meta name="twitter:creator" content="@leggetter" />
+</head>
+
+import tigrisConfig from "@site/tigris.config.js";
+
+export const CloudLink = ({ text = "Tigris Cloud" }) => {
+  return <a href={tigrisConfig.signupUrl}>{text}</a>;
+};
+
+Database search is okay, at best. To build awesome search experiences you need a
+dedicated search product.
+
+Earlier this month we
+[released Tigris Search in BETA](/blog/tigris-database-search-sync-beta/) and
+covered how it supports automatic syncronization between Tigris Database and
+Tigris Search, which improves developer productivity through enabling the team
+to focus on shipping features and not setting up and managing database-to-search
+syncronization infrastructure.
+
+But, you many not yet be in the position to migrate all your data over to Tigris
```

Comment:
However, you may be looking to quickly enhance your application experience by adding the capability to search existing data without waiting to migrate your database. Or, your data may not be in a database; maybe you've got all your data in S3.